### DISCUSSION TITLE
问题，视频播放BUG

### DISCUSSION THREAD

**Discussion by zhangjx075** [Richasy/Bili.Uwp#852] [问答]
我每次打开软件，点进去的第一个视频可以正常播放，但是之后的视频都没有画面，视频的声音，弹幕都正常没有问题（如图）。发现可以通过反复切换清晰度让画面重新出现。有一样问题的小伙伴吗？该怎么解决呢？解码器都安装了不是解码的问题。
<img width="717" alt="屏幕截图 2022-03-20 002105" src="https://user-images.githubusercontent.com/101015401/159129621-0e25f8ef-5ba7-4a9b-aafd-c8bf08e29034.png">

**Comment by Richasy** [comment]
是否尝试过切换其它的解码器？即在 `设置->播放器设置->解码偏好` 中将解码器切换到AV1或HEVC？

**Reply by zhangjx075** [reply]
尝试过的，AV1，HEVC，AVC都是打开软件的第一个视频能播放，后面的视频，画面放不出来，只有声音弹幕      /(ㄒoㄒ)/~~

**Reply by AC-wang** [reply]
> 是否尝试过切换其它的解码器？即在 `设置->播放器设置->解码偏好` 中将解码器切换到AV1或HEVC？
我是今天才下载使用的，如果没有安装解码器的话确实视频会出现只有声音弹幕没有画面的情况，但是没有出现楼主的所述的问题，而我的问题是在直播中，直播清晰度切换到蓝光和原画则是只有声音弹幕没有画面。

**Comment by Yan-nian** [comment]
我也同样有这个问题 显卡是3060

**Reply by zhangjx075** [reply]
我也是，笔记本的3060。混合输出和独显直连都会遇到这个问题
